lib/helpers.bash

```diff
@@ -789,8 +790,9 @@ function _disable-thing() {
 
 	_bash-it-clean-component-cache "${file_type}"
 
-	if [[ "$file_entity" = "all" ]]; then
-		printf '%s\n' "$file_entity $(_bash-it-pluralize-component "$file_type") disabled."
+	if [[ "$file_entity" == "all" ]]; then
+		_bash-it-component-pluralize "${file_type}"
```

Comment:
why not simply fix the function name?

Reply:
- actually, I think this line may have a typo. I'll double check in a few mins. 
- the function now returns the result in a variable, thus eliminating the subshell.

Reply:
@NoahGorny, I just pushed a slight change which may make this line read more clearly.